modules/viewing-cluster-capabilities.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="viewing_the_cluster_capabilities_{context}"]
+= Viewing the cluster capabilities
+As a cluster administrator, you can view the capabilities by using the `clusterversion` resource status.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the link:https://stedolan.github.io/jq/[jq package].
+
+.Procedure
+
+* To view the status of the cluster capabilities, run the following command:
++
+[source,terminal]
+----
+$ oc get -o json clusterversion version | jq '{spec: (.spec | {capabilities}), status: (.status | {capabilities})}'
```

Comment:
@darshan-nagaraj Same as above regarding jq link, please.

Reply:
This Jsonpath works on my end:
```
oc get clusterversion version -o jsonpath='{.spec.capabilities}{"\n"}{.status.capabilities}{"\n"}'
```

The result is like:
```
{"additionalEnabledCapabilities":["openshift-samples"],"baselineCapabilitySet":"None"}
{"enabledCapabilities":["openshift-samples"],"knownCapabilities":["baremetal","marketplace","openshift-samples"]}
```

Reply:
fixed.